web/templates/breadcrumbs.html

```
@@ -1,11 +1,11 @@
-{{ define "breadcrumbs" }}<nav id="breadcrumbs" class="flex-1 flex items-center gap-1 text-sm text-gray-500 overflow-hidden whitespace-nowrap">
-  <a class="text-blue-600 no-underline hover:underline" href="/browse/" aria-label="Home">&#8962;</a>
-  {{ range . }}
+{{ define "breadcrumbs" }}<nav class="flex-1 flex items-center gap-1 text-sm text-gray-500 overflow-hidden whitespace-nowrap">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)